PhysicsTools/PatAlgos/python/producersLayer1/metProducer_cfi.py

```diff
@@ -49,7 +49,7 @@
     computeMETSignificance  = cms.bool(False),
     # significance computation parameters, not used
     # if the significance is not computed
-    srcJets = cms.InputTag("selectedPatJets"),
+    srcJets = cms.InputTag("cleanedPatJets"),
```

Comment:
@davidlange6 that change is validated for the MET group, waiting for @monttj red/green light on the explanation provided